attestation-verifier/src/pkg/lib/common/crypt/certificate_store.go

```diff
@@ -134,16 +135,40 @@ func (cs *CertificatesStore) AddCertificatesToStore(certType, certFile string, c
 	defer defaultLog.Trace("crypt/certificate_store:AddCertificatesToStore() Leaving")
 
 	certStore := (*cs)[certType]
-	// Save certificate to file with common name
-	certPath := certStore.CertPath + strings.Replace(certFile, " ", "", -1) + ".pem"
+	safeDirs := []string{
```

Comment:
Need to check for authservice service also.

Reply:
i'll confirm again in the VM env, last I check I don't remember seeing authservice installing any certs

Reply:
added authservice cert path to the list